common_test.go

```diff
@@ -4,22 +4,54 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
-type SimpleLinguistTestSuite struct {
+const linguistURL = "https://github.com/github/linguist.git"
+
+type EnryTestSuite struct {
 	suite.Suite
+	repoLinguist string
+}
+
+func TestEnryTestSuite(t *testing.T) {
+	suite.Run(t, new(EnryTestSuite))
+}
+
+func (e *EnryTestSuite) SetupSuite() {
```

Comment:
`ts` is probably a better name for a testsuite, and wouldn't have required this name change because project name's changed